autoversioner/version/nonsemanticversion.go

```diff
@@ -0,0 +1,131 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package version
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	// ErrInvalidNonSemanticVer is returned if a version is found to be invalid when
+	// being parsed.
+	ErrInvalidNonSemanticVer = fmt.Errorf("Invalid NonSemantic Version")
+
+	// ErrInvalidDate is returned if the date being parsed is in a different
+	// format than expected.
+	ErrInvalidDate = fmt.Errorf("Invalid date format")
+
+	// ErrEmptyString is returned when an empty string is passed in for parsing.
+	ErrEmptyString = errors.New("Version string empty")
+
+	// ErrInvalidCharacters is returned when invalid characters are found as
+	// part of a version
+	ErrInvalidCharacters = errors.New("Invalid characters in version")
+)
+
+const (
+	DateFormat = "20060102"
+)
+
+// NonSemanticVer is for packages that follow nonsemantic version release
+type NonSemanticVer struct {
```

Comment:
can you update references to 'buildid' and 'nonsemanticver' to 'version' or similar?

Reply:
changed nonsemanticver to version